geonode/upload/tests/end2end/test_end2end.py

```diff
@@ -464,18 +465,23 @@ def test_import_3dtiles_overwrite(self):
 class ImporterWMSImportTest(BaseImporterEndToEndTest):
     @mock.patch.dict(os.environ, {"GEONODE_GEODATABASE": "test_geonode_data", "ASYNC_SIGNALS": "False"})
     @override_settings(GEODATABASE_URL=f"{geourl.split('/geonode_data')[0]}/test_geonode_data", ASYNC_SIGNALS=False)
-    @skip("To be updated without using development demo as source for the test")
     def test_import_wms(self):
         _, wms = WebMapService(
-            "https://development.demo.geonode.org/geoserver/ows?service=WMS&version=1.3.0&request=GetCapabilities"
+            f"{os.getenv('GEOSERVER_LOCATION')}/ows?service=WMS&version=1.3.0&request=GetCapabilities"
         )
         resource_to_take = next(iter(wms.contents))
         res = wms[next(iter(wms.contents))]
+
+        import urllib.parse
+
+        url = urllib.parse.quote(
+            f"{os.getenv('GEOSERVER_LOCATION')}ows?service=WMS&version=1.3.0&request=GetCapabilities", safe=""
+        )
         payload = {
-            "url": "https://development.demo.geonode.org/geoserver/ows?service=WMS&version=1.3.0&request=GetCapabilities",
+            "url": f"http://localhost:8000/proxy?url={url}",
```

Comment:
NOTE: the django serializer raises error if the url povided of the remote service is `geoserver:8080` since is not a valid URL, in order to make the test pass, we need to pass via the proxy since `localhost:8000` is a valid url

Reply:
@mattiagiupponi, we have `localhost:8000/gs` procsy, which is specific to Geoserver. Can't we use it?